influxdb/files/influxdb.conf

```diff
@@ -187,7 +187,11 @@ reporting-disabled = false
 [http]
   enabled = true
   bind-address = "{{ server.http.bind.address }}:{{ server.http.bind.port }}"
+  {%- if server.admin_user is defined %}
```

Comment:
So maybe extend admin structure in this way:
```
admin:
  enabled: true
  bind:
    address: 0.0.0.0
    port: 8083
  user:
    name: admin
    password: xyz
```

If we want to support multiple users (or consider adding that in future), it can be a list or better dict like this:
```
admin:
  user:
    admin:
      password: xyz
    admin2:
      password: xyz
```